crates/runtime/src/dataconnector/databricks.rs

```diff
@@ -339,6 +296,10 @@ const PARAMETERS: &[ParameterSpec] = &[
         .required()
         .secret()
         .description("The endpoint of the Databricks instance."),
+    ParameterSpec::component("sql_warehouse_id")
+        .required()
+        .secret()
+        .description("The ID of the SQL Warehouse. Only valid when mode is sql_warehouse"),
```

Comment:
```suggestion
        .description("The SQL Warehouse ID to use when 'mode' is set to 'sql_warehouse'"),
```

Reply:
Below for 'mode':

"The execution mode for running queries: 'spark_connect', 'delta_lake', or 'sql_warehouse'."